docs/content/how-do-i-write-policies.md

```diff
@@ -1396,11 +1347,18 @@ allow {
 }
 ```
 
-When the `allow` document is queried, the return value will be either `true` or `false`.
+When the `allow` document is queried with some input, the return value will be either `true` or `false`.
```

Comment:
Nit: don't need to qualify "with some input" here. The allow value will always be defined due to the default keyword.